damusTests/ReplyTests.swift

```diff
@@ -397,7 +397,8 @@ class ReplyTests: XCTestCase {
     
     func testParsePostUriPubkeyReference() throws {
         let id = "6fec2ee6cfff779fe8560976b3d9df782b74577f0caefa7a77c0ed4c3749b5de"
-        let parsed = parse_post_blocks(content: "this is a nostr:p:\(id) event mention")
```

Comment:
@jb55 Was there a reason why we supported `nostr:p:hex` and `nostr:e:hex` URIs? It doesn't seem to be to spec according to NIP-21.
https://github.com/nostr-protocol/nips/blob/master/21.md

Reply:
there was no spec when I wrote that